<ui version="4.0" >
 <class>installWindow</class>
 <widget class="QMainWindow" name="installWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Paketler Kuruluyor - Anatolya Kurulum</string>
  </property>
  <property name="styleSheet" >
   <string notr="true" >#installWindow { 
	background-image: url(:/artalan/artalan.png); }</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" name="gridLayout" >
    <item row="3" column="2" >
     <widget class="QPushButton" name="nextButton" >
      <property name="text" >
       <string>İleri</string>
      </property>
      <property name="icon" >
       <iconset resource="../../images/anatolyaimages.qrc" >
        <normaloff>:/icons/go-next.png</normaloff>:/icons/go-next.png</iconset>
      </property>
      <property name="flat" >
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3" >
     <spacer name="verticalSpacer" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType" >
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>20</width>
        <height>75</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="3" >
     <widget class="QGroupBox" name="installInfoGroup" >
      <property name="title" >
       <string/>
      </property>
      <property name="flat" >
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" >
       <item row="1" column="0" >
        <spacer name="verticalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>250</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="8" column="0" colspan="2" >
        <widget class="QProgressBar" name="installBar" >
         <property name="value" >
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Kurulan paket:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Kategori:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QLabel" name="packageLabel" >
         <property name="text" >
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" >
        <widget class="QLabel" name="categoryLabel" >
         <property name="text" >
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Toplam ilerleme:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2" >
        <widget class="Line" name="line" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2" >
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Paketler kuruluyor...
Bu işlem biraz zaman alabilir.
Şimdi kendinize bir kahve yapabilir ve Truvanızın kurulmasını bekleyebilirsiniz.</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLabel" name="randomPicture" >
         <property name="text" >
          <string/>
         </property>
         <property name="pixmap" >
          <pixmap resource="../../images/anatolyaimages.qrc" >:/betizler/installlogo.png</pixmap>
         </property>
        </widget>
       </item>
       <item row="5" column="1" >
        <spacer name="horizontalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>300</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1" >
        <widget class="QLabel" name="infoLabel" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" >
     <spacer name="horizontalSpacer" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1" >
     <widget class="QPushButton" name="backButton" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="text" >
       <string>Geri</string>
      </property>
      <property name="icon" >
       <iconset resource="../../images/anatolyaimages.qrc" >
        <normaloff>:/icons/back.png</normaloff>:/icons/back.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../images/anatolyaimages.qrc" />
 </resources>
 <connections/>
</ui>
